R/execution.r

```diff
@@ -25,6 +25,18 @@ plot_builds_upon <- function(prev, current) {
     lcurrent >= lprev && identical(current[[1]][1:lprev], prev[[1]][1:lprev])
 }
 
+ask <- function(prompt = '') {
+    answer <- NA
+    while (is.na(answer)) {
+        answer <- switch(readline(prompt),
```

Comment:
yeah, `quit()` mainly makes sense in the context of `jupyter (qt)console --kernel=ir`